contracts/cw1-subkeys/README.md

```diff
@@ -46,12 +53,22 @@ It also adds one more query type:
 enum QueryMsg {
     Allowance {
         spender: HumanAddr,
-    }
+    },
+    AllAllowances {
+        start_after: Option<HumanAddr>,
+        limit: Option<u32>,
+    },
 }
 
-pub struct AllowanceResponse {
```

Comment:
You removed the `AllowanceResponse`? I think you need both.

Oh, this is README....

Reply:
I wanted to make helpers.ts and msg.rs naming identical